inst/example/ui-helpers.R

```diff
@@ -14,7 +14,7 @@ data <- data.frame(
   content = c("Item one", "Item two",
               "Ranged item", "Item four"),
   start   = c("2016-01-10", "2016-01-11",
-              "2016-01-20", "2016-02-14 15:00:00"),
+              "2016-01-20", "2016-02-14"),
```

Comment:
is there a specific reason the time of day here was removed?

Reply:
a) i think this also works with 4.16.1.

b) the code in the example didn't match with the code in the example.  err, i.e., what was displayed to the user as the code included the time of day, but the code that generated the timeline displayed to the user did *not* include the time of day.  (modulo my being brain-dead.)